plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveOnDataLake.java

```diff
@@ -66,7 +66,11 @@
 public abstract class BaseTestHiveOnDataLake
         extends AbstractTestQueryFramework
 {
-    private static final String HIVE_TEST_SCHEMA = "hive_datalake";
+    /**
+     * It's important to mix case here to detect if we properly handle rewriting
+     * properties between Trino and Hive (e.g for Partition Projection)
+     */
```

Comment:
This could be ordinary comment (`// ...`). It's not worded as a javadoc.

Reply:
Ok changed